### ISSUE TITLE
a error

### ISSUE THREAD

**Issue by IFRFSX** [srifqi/superflat#4]
```
2020-04-09 21:57:59: ERROR[Main]: ModError: Failed to load and run script from /home/bingfengfsx/Programs/minetest/bin/../mods/superflat/init.lua:
2020-04-09 21:57:59: ERROR[Main]: Unknown node: flowers:dandelion_white
2020-04-09 21:57:59: ERROR[Main]: stack traceback:
2020-04-09 21:57:59: ERROR[Main]: 	[C]: in function 'get_content_id'
2020-04-09 21:57:59: ERROR[Main]: 	...x/Programs/minetest/bin/../mods/superflat/decoration.lua:330: in main chunk
2020-04-09 21:57:59: ERROR[Main]: 	[C]: in function 'dofile'
2020-04-09 21:57:59: ERROR[Main]: 	...fengfsx/Programs/minetest/bin/../mods/superflat/init.lua:16: in main chunk
```

EDIT by srifqi: formatting

**Comment by srifqi** [comment]
Does enabling `flowers` mod solve that error? It seems like there is no check for availability of `flowers` mod (from Minetest Game).
https://github.com/srifqi/superflat/blob/02c17fa8f84bf42962ba496b7143d1f8ca16eebe/decoration.lua#L330-L336

**Comment by IFRFSX** [comment]
@srifqi 
I've always enabled it. Maybe it's caused by the new minetest game code change?
https://github.com/minetest/minetest_game/blob/master/mods/flowers/init.lua

**Comment by IFRFSX** [comment]
#5
please marge this!

**Comment by srifqi** [comment]
Fixed in #5.
